### PR TITLE
Bump to Go version 1.13.3

### PR DESCRIPTION
**Pull request by drewvanstone**
## High-level description

This PR bumps the version of Go in the package panda build container from 1.12.9 to 1.13.3. The new version of Go has useful features for error handling that we'd like to start taking advantage of. 

## Corresponding DC/OS tickets (required)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by drewvanstone** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by janisz** [comment]
@mesosphere-mergebot changelog-not-required not a customer facing change

**Comment by janisz** [comment]
@philipnrmn @branden Can you help with telegraf problem?

File: packages/telegraf/windows.build.ps1 [hunk 1]
```diff
@@ -12,5 +12,6 @@ $Env:TMP="C:\tmp"
 Set-Location C:\gopath\src\github.com\influxdata\telegraf\
 $env:GOOS="windows"
 $env:GOARCH="386"
+$env:GOPATH="c:\gopath"
```
Comment (janisz) [inline packages/telegraf/windows.build.ps1]:
```suggestion
```

Comment (drewvanstone) [reply packages/telegraf/windows.build.ps1]: Why should this be unset?

Comment (drewvanstone) [reply packages/telegraf/windows.build.ps1]: Looks like Windows build just passed

Comment (janisz) [reply packages/telegraf/windows.build.ps1]: I set this because I though it might solve the issue with windows build.

**Pull request merged**